modules/cudf/src/column.ts

```diff
@@ -700,6 +701,98 @@ export interface Column<T extends DataType = any> {
    */
   isNotNaN(memoryResource?: MemoryResource): Column<Bool8>;
 
+  /**
+   * Creates a column of `BOOL8` elements indicating strings in which all characters are valid for
```

Comment:
Maybe I'm misunderstanding, but, `indicating strings`?

Reply:
I tried to keep the verbiage similar to out other usage, but the libcudf doc this was cribbed from reads
> Returns a boolean column identifying strings in which all characters are valid for conversion to floats.

Do you think that "identifying" would be more clear than "indicating"? These are basically synonymous usage in my mind but also happy to change

Reply:
Chatted with Bryan about this, this is fine.

Reply:
We seem to use "indicating" pretty consistently throughout already in similar bool returns so I'll keep this for now to be consistent but maybe we can/should change across the board.